realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -353,6 +380,27 @@ public Builder(Context context) {
             initializeBuilder(context.getFilesDir());
         }
 
+        /**
+         * Creates an instance of the Builder for the RealmConfiguration.
+         *
+         * This will use the app's own internal directory for storing the Realm file. This does not require any
+         * additional permissions. The default location is {@code /data/data/<packagename>/files}, but can
+         * change depending on vendor implementations of Android.
+         *
+         * Initial dataset will be imported from the provided Realm asset file.
```

Comment:
I would rephrase this to: 

When opening the Realm for the first time, instead of creating an empty file, the Realm file will be copied from the provided assets file and used instead. WARNING: This could potentially be a lengthy operation so should ideally be done on a background thread.